### PR TITLE
🌐 Add Korean translation for Index

### PR DESCRIPTION
**Pull request by hard-coders**
This is the first translation for Korean Language page(see #2017), `index.md`.
I hope lots of developers who are Korean participate in this community.😁

And.. please review the translation, thanks.

### CODE REVIEW

**Review by jrim — APPROVED** [review]
It is absolutely well-translated. 너무 좋습니다!

File: docs/ko/docs/index.md [hunk 1]
```diff
@@ -279,175 +275,175 @@ def update_item(item_id: int, item: Item):
...
+...그리고 단 하나의 선언으로 여러분이 얻는 것은:
 
-* Editor support, including:
-    * Completion.
-    * Type checks.
-* Validation of data:
-    * Automatic and clear errors when the data is invalid.
-    * Validation even for deeply nested JSON objects.
-* <abbr title="also known as: serialization, parsing, marshalling">Conversion</abbr> of input data: coming from the network to Python data and types. Reading from:
+* 다음을 포함한 에디터 지원:
```
Comment (hard-coders) [inline docs/ko/docs/index.md:328]:
```suggestion
* 다음을 포함한 편집기 지원:
```

File: docs/ko/docs/index.md [hunk 2]
```diff
@@ -279,175 +275,175 @@ def update_item(item_id: int, item: Item):
...
 
-...to:
+...으로:
 
 ```Python
         ... "item_price": item.price ...
 ```
 
-...and see how your editor will auto-complete the attributes and know their types:
+...그러고 나서 여러분의 에디터가 속성과 타입을 알고 자동 완성하는지 보세요:
```
Comment (hard-coders) [inline docs/ko/docs/index.md:395]:
```suggestion
...그러고 나서 여러분의 편집기가 속성과 타입을 알고 자동 완성하는지 보세요:
```

**Review by hard-coders — COMMENTED** [review]
- Replace "에티더" with "편집기" when translating "editor"
- Add Sponsor section from a42c69049647f4fb731afc86c6eeb959248217e4

**Review by swnim — APPROVED** [review]

File: docs/ko/docs/index.md [hunk 3]
```diff
@@ -293,175 +289,175 @@ def update_item(item_id: int, item: Item):
...
-And now, go to <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>.
+그리고 이제, <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>로 이동합니다.
 
-* The alternative documentation will also reflect the new query parameter and body:
+* 대안 문서 역시 새 쿼리 매개변수와 본문을 반영합니다:
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-06-redoc-02.png)
 
-### Recap
+### 톺아보기
```
Comment (hard-coders) [inline docs/ko/docs/index.md:318]:
```suggestion
### 요약
```

**Comment by tiangolo** [comment]
This is awesome! :tada: :rocket: 

Thanks a lot @hard-coders for all the work :nerd_face: :cake: 

And thanks @jrim and @swnim for the reviews :bow: :cookie:

**Pull request merged**